adminer/include/editing.inc.php

```diff
@@ -290,7 +290,6 @@ function edit_fields($fields, $collations, $type = "TABLE", $foreign_keys = arra
 * @return bool
 */
 function process_fields(&$fields) {
-	ksort($fields);
```

Comment:
Unfortunately if you ksort the fields after reordering with javascript it'll put them right back where they were. An alternative would be to add a hidden "Position" input and tweak that with javascript, then sort based on that. Should I write that up?